Resources/public/js/views/tabs/ez-locationviewversionstabview.js

```diff
@@ -202,26 +204,65 @@ YUI.add('ez-locationviewversionstabview', function (Y) {
         },
 
         /**
+         * Returns the selected archived versions as VersionInfo
+         *
+         * @method _getSelectedArchived
+         * @protected
+         * @return {Array} of eZ.VersionInfo
+         */
+        _getSelectedArchived: function () {
```

Comment:
it's almost the same as `_getSelectedDrafts`, you can probably reorganize a bit the code to use the same method